GVFS/GVFS.UnitTests/Platform.Mac/MacFileSystemVirtualizerTests.cs

```diff
@@ -90,6 +90,7 @@ public void UpdatePlaceholderIfNeeded()
             const string UpdatePlaceholderFileName = "testUpdatePlaceholder.txt";

```

Comment:
Are there any unit tests that can be added for the changes to `MacFileSystemVirtualizer`?

Reply:
Added an issue to track these changes. #1365